site/docs/master/install-overview.md

```diff
@@ -1,46 +1,112 @@
-# Install overview
+# Install Overview
 
-You can run Velero in clusters on a cloud provider or on-premises. For detailed information, see our list of [supported providers][0].
+- [Prerequisites](#prerequisites)
+- [Install the command-line interface (CLI)](#install-the-cli)
+- [Install and configure the server components](#install-and-configure-the-server-components)
+- [Advanced installation topics](#advanced-installation-topics)
 
-We strongly recommend that you use an [official release][1] of Velero. The tarballs for each release contain the
-`velero` command-line client.
+## Prerequisites
+- access to a Kubernetes cluster, v1.TODO or later, with DNS and container networking enabled.
```

Comment:
In spirit, the content of this section and the content of the page `Requirements` are the same. I think this section should be moved there. Alternatively, the content of that page could be added here and that page deleted.

Reply:
Yep, good point - I decided to move the content into this page since it seemed to fit best if split up. I also added a sub-TOC to the "advanced topics" section to help with navigation there.